MekHQ/src/mekhq/AtBGameThread.java

```diff
@@ -395,7 +409,7 @@ private void configureBot(BotClient botClient, BotForce botForce) {
                 botClient.getLocalPlayer().setTeam(botForce.getTeam());
                 botClient.getLocalPlayer().setStartingPos(botForce.getStart());
 
-                if (botForce.getCamoCategory().equals(Player.NO_CAMO)) {
+                if (botForce.getCamoCategory() == Player.NO_CAMO) {
```

Comment:
Why was this swapped from .equals? That makes no sense

Reply:
Couldn't say. I suspect it was introduced from a merge of the latest Master.